### ISSUE TITLE
TrailingSemicolon warning for @include blocks

### ISSUE THREAD

**Issue by steffenweber** [brigade/scss-lint#198]
```
$ cat test.scss 
body {
  @include foo {
    background: #f00;
  }
}
$ scss-lint test.scss 
test.scss:2 [W] TrailingSemicolon: Declaration should not have a space before the terminating semicolon
$ scss-lint -version
scss-lint 0.26.0
```

I guess this warning should only be generated for `@include` statements without curly braces, right? Adding a semicolon after the closing curly brace in line 4 does not make the warning disappear.
